client/src/modules/email-report/modal/liste-categories-people-ctrl.js

```diff
@@ -0,0 +1,30 @@
+angular.module('bhima.controllers')
+    .controller('CategoriesPeopleModalController', CategoriesPeopleModalController);
+
+CategoriesPeopleModalController.$inject = [
+    'CategoriesPeopleService', 'SessionService', 'util',
+    'NotifyService', 'ScrollService', 'bhConstants', 'uiGridConstants',
+];
+
+/**
+ * Report group Controller
+ *
+ * @description
+  The controller that collect every definied group of people 
+  where the user can select a profile
+ */
+
+function CategoriesPeopleModalController(CategoriesPeople, Session, util, Notify, ScrollTo, bhConstants, uiGridConstants) {
```

Comment:
The comment says this is `Report Group Controller`, but the name says `CategoriesPeopleModalController`.  What is it?